adapter/denyChecker/adapter.go

```diff
@@ -15,59 +15,29 @@
 package denyChecker
 
 import (
-	"fmt"
-
 	"github.com/golang/protobuf/proto"
 	"google.golang.org/genproto/googleapis/rpc/code"
 
 	"istio.io/mixer/pkg/aspect/denyChecker"
 	"istio.io/mixer/pkg/aspectsupport"
 )
 
-const (
-	// ImplName is the canonical name of this implementation
-	ImplName = "istio/denyChecker"
-)
-
-// Register records the existence of this adapter
 func Register(r aspectsupport.Registry) error {
 	return r.RegisterDeny(newAdapter())
 }
 
 type adapterState struct{}
 
-func newAdapter() denyChecker.Adapter {
-	return &adapterState{}
-}
-
-func (a *adapterState) Name() string {
-	return ImplName
-}
-
-func (a *adapterState) Description() string {
-	return "Deny every check request"
-}
+func newAdapter() denyChecker.Adapter                          { return &adapterState{} }
+func (a *adapterState) Name() string                           { return "istio/denyChecker" }
```

Comment:
At this point, I don't see a need to have the names and descriptions as constants since we are registering the adapters systematically. So the names can be found by looking at the registered adapters.